src/product/dashboard.njk

```diff
@@ -14,8 +14,10 @@ meta:
             <h4 class="font-light mt-2 text-gray-500">
                 Build visually appealing and useful UIs in minutes!
             </h4>
-            <div class="justify-center items-center m-auto w-full flex mt-10">
-                {% image "../images/dashboard/dashboard.svg", "FlowFuse Dashboard", [1920] %}
+            <div class="justify-center items-center m-auto w-full flex mt-10 gap-3 dashboard-image">
+                {% image "../images/dashboard/bar-chart.gif", "bar chart",[1920]%}
```

Comment:
`[1920]%}`

That's meant to set the final size of the images. The entire image with the 3 charts was 1920px, but 1024px would have been enough. You should add the size of each chart individually now that each one is a separate file.